concepts/05 UI Components/DataGrid/70 Custom Keyboard Navigation/10 Focus Cells Programatically.md

```diff
@@ -0,0 +1,7 @@
+You can use the following methods to focus a specific cell:
```

Comment:
Does the word "specific" adds anything of value in this scenario? That is, I mean if it means anything else than just "a cell"?

```suggestion
You can call the following methods to focus a cell:
```

Reply:
Not really. I just wanted to emphasize that it's the cell that a customer chooses.

Reply:
Well, that could have been any cell, and we cannot narrow it down any further, like "target cell" because we speak of a hypothetical method call that haven't occurred in reality just yet. In theory, we could have said "any required cell". 

However, this would have meant the same as "any cell" or just "a cell" in the end.